readthedocs/search/tests/test_proxied_api.py

```diff
@@ -22,5 +22,5 @@ def test_headers(self, api_client, project):
         }
         resp = self.get_search(api_client, search_params)
         assert resp.status_code == 200
-        cache_tags = f'{project.slug},{project.slug}-{version.slug},{project.slug}-rtd-search'
+        cache_tags = f'{project.slug},{project.slug}:{version.slug},{project.slug}:rtd-search'
```

Comment:
We could also change the `:rtd-search`/:rtd-footer` part of the tag to something like

`:-rtd-search` or `:-rtd-search` or anything that wouldn't collide with a version name.
The collision can happen only inside the same project, so it's less worrisome (if the project has a version named `rtd-search`).